### PR TITLE
Update EIP-747: Add myself as an author and move back to Review

### PR DESCRIPTION
**Pull request by Pandapip1**
Will need approval from one of Dan Finlay (@danfinlay) or Esteban Mino (@estebanmino)

More changes will likely be needed to bring this up to EIP-1

### CODE REVIEW

**Review by danfinlay — APPROVED** [review]

File: EIPS/eip-747.md [hunk 1]
```diff
@@ -105,61 +117,15 @@ ethereum.request({
...
-
-This phenomenon is exacerbated by the trend towards airdropped tokens, which has been a cause of network congestion, because spamming people with new tokens has so far been rewarded with user attention.
-
-While some people might suggest we begin a TCR of trusted tokens to watch, this would not solve the client-side bandwidth issues, nor the airdropped token spam issues. What we really want is a small list of tokens the user cares about.
-
-Most of the time a user is adding a asset, they learned about it on a website. At that moment, there is a natural alignment of interests, where the website wants the user to track their asset, and the user wants to track it. This is a natural point to introduce an API to easily allow these parties to collaborate, without involving the politics of the wallet's developers.
+## Security Considerations
 
-## Implementation
+### SSRF
```
Comment (Pandapip1) [inline EIPS/eip-747.md:126]: See https://www.youtube.com/watch?v=RCJdPiogUIk for an explanation as to why this is an issue

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@danfinlay mind re-approving?

**Review by danfinlay — APPROVED** [review]

**Comment by danfinlay** [comment]
done

**Pull request merged**